docs/sources/auth/generic-oauth.md

```diff
@@ -206,9 +206,10 @@ allowed_organizations =
     allow_sign_up = true
     client_id = <OpenID Connect Client ID from Centrify>
     client_secret = <your generated OpenID Connect Client Sercret"
-    scopes = openid email name
+    scopes = openid profile email
```

Comment:
why this change? this changes the example config of this oauth provider

Reply:
This change changes the example config so that it actually works :) - We're using Centrify (now Idaptive), and it breaks with the example config. After a lot of messing around, "openid profile email" worked for us. We could also use just email, but this would mean new users did not have name field filled.

Reply:
PS not that clear, but you can see from the examples that "name" is not an available scope: https://developer.idaptive.com/docs/openid-connect-_new

Reply:
ah, good! could have mentioned that in the description :) Got confused as the rest of the changes where of such different type

Reply:
That would have made a lot more sense, sorry, next time! :)